nano/node/node.cpp

```diff
@@ -2453,6 +2453,32 @@ void nano::node::block_confirm (std::shared_ptr<nano::block> block_a)
 	}
 }
 
+void nano::node::confirm_frontiers ()
+{
+	// Limit maximum count of elections to start. max_broadcast_queue for regular nodes, x10 for representatives
+	bool representative (config.enable_voting && wallets.reps_count > 0);
+	size_t max_elections (representative ? active.max_broadcast_queue * 10 : active.max_broadcast_queue);
+	size_t elections_count (0);
+	auto transaction (store.tx_begin_read ());
+	for (auto i (store.latest_begin (transaction)), n (store.latest_end ()); i != n && elections_count < max_elections; ++i)
+	{
```

Comment:
I wonder if this will create some unnecessary traffic if it asks for confirmations for accounts which may get implicitly confirmed in a previous iteration from another account?
Do you think it would make sense to have a separate table (or something) which keeps track of any accounts which aren't fully confirmed rather than scan all 600k+ accounts every time? Most accounts are generally static I guess.
`block_confirm` calls `network.broadcast_confirm_req (block_a);` is that necessary here as well?

Reply:
`block_confirm` replaced with `active_transactions::start`

Reply:
Hm, I guess we can disable check for bootstraps that didn't process any blocks (most bootstraps for synced node)